_episodes/01-intro.md

```diff
@@ -79,17 +62,102 @@ the Bourne Again SHell
 Bash is the default shell on most modern implementations of Unix
 and in most packages that provide Unix-like tools for Windows.
 
-### Why bother?
-Using Bash or any other shell
-sometimes feels more like programming than like using a mouse.
-Commands are terse (often only a couple of characters long),
-their names are frequently cryptic,
-and their output is lines of text rather than something visual like a graph.
-On the other hand,
-with only a few keystrokes, the shell allows us to combine existing tools into 
-powerful pipelines and handle large volumes of data automatically. This automation
-not only makes us more productive but also improves the reproducibility of our workflows by 
-allowing us to repeat them with a few simple commands.
+### What does it look like?
```

Comment:
I like moving the terminology like prompt, command, flag etc. here from the navigating files and directories episode. I think it makes a lot more sense in this ep.